geanyvc/src/geanyvc.c

```diff
@@ -1586,17 +1591,25 @@ vccommit_activated(G_GNUC_UNUSED GtkMenuItem * menuitem, G_GNUC_UNUSED gpointer
 	gtk_paned_set_position(GTK_PANED(vpaned2), height * 50 / 100);
 
 #ifdef USE_GTKSPELL
-	speller = gtkspell_new_attach(GTK_TEXT_VIEW(messageView), NULL, &spellcheck_error);
+    #if GTK_CHECK_VERSION (3, 0, 0)
+        speller = gtk_spell_checker_new ();
+        gtk_spell_checker_attach (speller, GTK_TEXT_VIEW (messageView));
+    #else
+        speller = gtkspell_new_attach(GTK_TEXT_VIEW(messageView), NULL, &spellcheck_error);
+    #endif
```

Comment:
…though here it might be better to do the contrary and add a compat function for GTK3
```C
static GtkSpellChecker *gtkspell_new_attach(GtkTextView *view, const gchar *lang, GError **error)
{
    GtkSpellChecker *speller = gtk_spell_checker_new ();
    gtk_spell_checker_attach (speller, view);
    return speller;
}
```
next to the other compat macros so there is a single gtkspell compat layer in one single location?